front/src/modules/rollingStock/components/rollingStockEditor/RollingStockEditorCurves.tsx

```diff
@@ -306,52 +441,94 @@ export default function RollingStockEditorCurves({
     selectedPowerRestriction,
   ]);
 
+  const { data: availableModes } = osrdEditoastApi.endpoints.getInfraVoltages.useQuery();
+
   return (
     <>
       {rollingstockParams && (
         <div className="d-flex flex-wrap px-3 pb-2">
-          <SelectorSNCF
-            key="comfort-level-selector"
-            borderClass="selector-blue"
-            title="comfortLevel"
-            itemsList={rollingstockParams.comfortlevels}
-            onItemSelected={dispatchComfortLvl}
-            selectedItem={selectedComfortLvl}
-            translationFile="rollingstock"
-            translationList="comfortTypes"
-          />
-          <SelectorSNCF
-            key="traction-mode-selector"
-            borderClass="selector-pink"
-            title="tractionMode"
-            itemsList={rollingstockParams.tractionModes}
-            onItemSelected={dispatchTractionMode}
-            selectedItem={selectedTractionMode}
-            translationFile="rollingstock"
-          />
+          <div className="selector-container">
+            <SelectorSNCF
+              key="comfort-level-selector"
+              borderClass="selector-blue"
+              title="comfortLevels"
+              itemsList={rollingstockParams.comfortLevels}
+              selectedItem={selectedComfortLvl}
+              hoveredItem={hoveredRollingstockParam}
+              onItemSelected={dispatchComfortLvl}
+              onItemHovered={setHoveredRollingstockParam}
+              onItemRemoved={removeAnotherRsParam}
+              translationFile="rollingstock"
+              translationList="comfortTypes"
+            />
+            <AddRollingstockParam
+              displayedLists={rollingstockParams}
+              updateDisplayedLists={updateComfortLevelsList}
+              allOptionsList={comfortOptions}
+              listName="comfortLevels"
+            />
```

Comment:
I think it would be nicer to have this component directly in the selector. You can create a meta props in the selector, for instance addParams={ displayedList, updatedDisplayedList, allOptionsList, listName}
```suggestion
            <AddRollingstockParam
              displayedLists={rollingstockParams}
              updateDisplayedLists={updateComfortLevelsList}
              allOptionsList={comfortOptions}
              listName="comfortLevels"
            />
```